### DISCUSSION TITLE
SQL Error 2656 when fetching table metadata dropdowns

### DISCUSSION THREAD

**Discussion by polybugger** [dbeaver/dbeaver#12282] [Q&A]
Hi,

I've installed the latest version of dbeaver community version (21.0.3), and downloaded the necessary vertica jar files. 
However, when I try to click on the Columns under a table, I keep getting this error:

SQL Error [2656] [42703]: [Vertica][VJDBC](2656) ERROR: Column com.child_object does not exist
  [Vertica][VJDBC](2656) ERROR: Column com.child_object does not exist

![dbeaver_VeghF2E2Oz](https://user-images.githubusercontent.com/14595449/116443940-d22a3e80-a886-11eb-9f30-b128553d2ca7.png)

I tried googling everywhere for hours on this error, but I'm not getting any definite answers.
I wonder if somebody else has encountered this error?
And how they addressed the issue would be very helpful.

Clicking on the other metadata folders also produce the same error (Unique Keys, Foreign Keys, References).
But executing an SQL query works fine though.

Sincerely,

**Comment by LonwoLonwo** [comment]
Hello @polybugger  
We already fixed this problem here #12127 , I hope.
You can test it in [Early Access version](https://dbeaver.io/files/ea/).

**Reply by polybugger** [reply]
Hi,

Wonderful!

I just downloaded the early access version in the link you provided, and I'm not getting the errors anymore.
Clicking on the metadata folders properly fetches the data and I can see the table columns list now.

Thank you very much!!

**Comment by polybugger** [comment]
Fixed in early access version 21.0.3
https://dbeaver.io/files/ea/
